src/structures/Guild.js

```diff
@@ -299,40 +301,44 @@ class Guild extends Base {
      */
     this.systemChannelFlags = new SystemChannelFlags(data.system_channel_flags).freeze();
 
-    /**
-     * The maximum amount of members the guild can have
-     * <info>You will need to fetch the guild using {@link Guild#fetch} if you want to receive this parameter</info>
-     * @type {?number}
-     * @name Guild#maximumMembers
-     */
-    if (typeof data.max_members !== 'undefined') this.maximumMembers = data.max_members || 250000;
+    if (typeof data.max_members !== 'undefined') {
+      /**
+       * The maximum amount of members the guild can have
+       * <info>You will need to fetch the guild using {@link Guild#fetch} if you want to receive this parameter</info>
+       * @type {?number}
```

Comment:
The field defaulting it to 250000.

```suggestion
       * @type {number}
```

Reply:
i'll remove the if statements surrounding these aswell then ye?

Reply:
upon further look - both of these are optional on the [API Docs](https://discord.com/developers/docs/resources/guild) however, `max_members` is sent in both gateway `GUILD_CREATE` and REST `GET /guilds/:id` - but im not sure if the docs are wrong here or there is a case where it can be omitted so i marked it as nullable to be safe - its also not marked as null on the docs so there isn't a need to apply a default to max_members - it should always be a number

Reply:
The `max_members` field wasn't present within the `GUILD_CREATE` event when it was initially rolled out (see https://github.com/discordjs/discord.js/pull/3168#discussion_r272542448) hence why `Guild#fetch()` was introduced along with the JSDoc info. But that seemed to have changed... and it is now present there, I agree with @NotSugden that they should still probably be marked as nullable for both `maximumMembers` & `maximumPresences` since that's how it is documented on the API docs and just in case Discord decides to make another change.